src/controller/python/OpCredsBinding.cpp

```diff
@@ -687,4 +720,24 @@ PyChipError pychip_GetCompletionError()
     return ToPyChipError(sTestCommissioner.GetCompletionError());
 }
 
+extern "C" {
+// Function to get the RCAC data from the sTestCommissioner
+void pychip_GetCommissioningRCACData(uint8_t ** rcacDataPtr, size_t * rcacSize)
```

Comment:
if you do uint8_t * const * rcacDataPtr, can you get rid of the const cast below?

Reply:
Better still, can you have the python side allocate space and copy out? That way you're not handing over the c++ memory to python.

Reply:
Hi Cecille,
I have updated the code to attempt to the best of my knowledge and ability to update allocating the memory in ChipDeviceCtrl python module's get_rcac function, then using memcpy in C++ I have copied the memory from C++ into the allocated python space.
Please let me know if this will work to resolve this